tests/unit/test_bisector.py

```diff
@@ -12,7 +12,13 @@
 from mozregression.errors import LauncherError, MozRegressionError
 
 
+def mock_abstractmethod():
+    raise NotImplementedError
+
+
 class TestBisectorHandler(unittest.TestCase):
+
+    @patch.multiple(BisectorHandler, __abstractmethod__=mock_abstractmethod())
```

Comment:
Instead of all this magic, why don't create a **TestBisectorHandler** new class, that inherit form **BisectorHandler** and implement the required methods ? Then the tests can use **TestBisectorHandler** instead of **BisectorHandler**.